test/e2e/specs/specs-playwright/wp-signup__screenshot.ts

```diff
@@ -0,0 +1,284 @@
+/**
+ * @group legal
+ */
+import fs from 'fs';
+import {
+	ChangeUILanguageFlow,
+	DataHelper,
+	LoginPage,
+	NavbarComponent,
+	PlansPage,
+	CartCheckoutPage,
+	SidebarComponent,
+	UserSignupPage,
+	BrowserManager,
+	DomainSearchComponent,
+} from '@automattic/calypso-e2e';
+import archiver from 'archiver';
+import FormData from 'form-data';
+import fetch from 'node-fetch';
+import { Page, Browser } from 'playwright';
+import type { LanguageSlug } from '@automattic/languages';
+
+const selectors = {
+	isWhiteLogin: '.is-section-login.is-white-login',
+	isBlueLogin: '.is-section-login:not( .is-white-login )',
+	isWhiteSignup: 'body.is-white-signup.is-section-signup',
+	isBlueSignup: '.is-section-signup:not( .is-white-signup ) .signup__step.is-user .signup-form',
+};
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'ToS acceptance tracking screenshots' ), function () {
+	const blogName = 'e2eflowtestingtos1.wordpress.com';
+	const cartItemForBusinessPlan = 'WordPress.com Business';
+	let page: Page;
+	let plansPage: PlansPage;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+	} );
+
+	describe( 'ToS signup, login, and checkout', function () {
+		jest.setTimeout( 1800000 );
+		let cartCheckoutPage: CartCheckoutPage;
+		const magnificientNonEnLocales = [
+			'pt-br',
+			'fr',
+			'es',
+			'de',
+			'he',
+			'ja',
+			'it',
+			'nl',
+			'ru',
+			'tr',
+			'id',
+			'zh-cn',
+			'zh-tw',
+			'ko',
+			'ar',
+			'sv',
+		];
+
+		it( 'Screenshot blue background signup page in desktop viewport, en and Mag-16 locales', async function () {
+			const userSignupPage = new UserSignupPage( page );
+			let domainSearchComponent: DomainSearchComponent;
+			const username = `e2eflowtestingdomainonly${ DataHelper.getTimestamp() }`;
+			for ( const locale of [ ...magnificientNonEnLocales, 'en' ] ) {
+				await userSignupPage.visit( { path: `domain/${ locale }` } );
+				await BrowserManager.setStoreCookie( page, { currency: 'JPY' } );
+
+				domainSearchComponent = new DomainSearchComponent( page );
+				await domainSearchComponent.search( username + '.live' );
+				await domainSearchComponent.selectDomain( '.live' );
+
+				await page.click(
+					'[data-e2e-type="domain"].site-or-domain__choice .site-or-domain__choice-button button'
+				);
+				await page.waitForSelector( selectors.isBlueSignup );
+
+				await page.screenshot( {
+					path: `tos_blue_signup_desktop_${ locale }.png`,
```

Comment:
I ran this script locally and turns out this would end up creating all screenshots at the `test/e2e` directory.

I'd suggest that in a follow-up issue this be addressed, so that all screenshots are stored in a directory, for example `test/e2e/tos-screenshots`.